src/main/java/teammates/ui/template/InstructorFeedbackSessionActions.java

```diff
@@ -34,6 +34,8 @@ public InstructorFeedbackSessionActions(PageData data, FeedbackSessionAttributes
                                             InstructorAttributes instructor) {
         String courseId = session.courseId;
         String feedbackSessionName = session.feedbackSessionName;
+        String returnUrl = isHome ? Const.ActionURIs.INSTRUCTOR_HOME_PAGE
+                                  : Const.ActionURIs.INSTRUCTOR_FEEDBACKS_PAGE;
```

Comment:
Actually, if we want extensibility, it's probably best to just use `returnUrl` from the start, with the `*PageData` calling such functions and passing in the URI rather than a boolean like `isHome`.